tests/unit/api_client/api_test_base.py

```diff
@@ -1,8 +1,8 @@
 from typing import Any, Callable
+from unittest.mock import Mock
```

Comment:
during this process I realized we were needlessly installing `mock`, as it has been a part of Python since 3.3.